content/integrations/syslog_ng.md

```diff
@@ -1,9 +1,28 @@
 ---
-title: Syslog-ng Log collection
-kind: Documentation
+title: Syslog-ng
+kind: integration
 description: "Configure Syslog-ng to gather logs from your host, containers & services."
+short_description: "Configure Syslog-ng to gather logs from your host, containers & services."
```

Comment:
Serial comma here and in the line above. Configure Syslog-ng to gather logs from your host, containers & services.